CONFIG.md

```diff
@@ -462,10 +463,18 @@ path: pools.<pool_name>.users.<user_index>.pool_size
 default: 9
 ```
 
-Maximum number of server connections that can be established for this user
+Maximum number of server connections that can be established for this user.
 The maximum number of connection from a single Pgcat process to any database in the cluster
 is the sum of pool_size across all users.
 
+### min_pool_size
+```
+path: pools.<pool_name>.users.<user_index>.min_pool_size
+default: 0
+```
+
+Minimum number of server connections to retain for this pool.
```

Comment:
This is actually the minimum number of idle connections to retain in the pool
https://github.com/postgresml/pgcat/issues/720

That was a TIL for me, the name of the config is misleading IMO but the least we could do is make the docs accurate.